python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -232,6 +231,40 @@ def __new__(
             has_uninstalled_deps=has_uninstalled_deps,
             manifest_preparer=manifest_preparer,
         )
-        if manifest_preparer:
-            manifest_preparer.on_load(val)
-        return val
+
+    @public
+    def prepare_if_dev(self) -> None:
+        """Prepare a dbt project at run time during development.
```

Comment:
We should be specific about what we mean by "during development". I.e. is it when `dagster dev` is used?